### PR TITLE
feat(labels): [BACK-1651, BACK-1639] Update `updateCollection` mutation to incorporate labels

### PR DESCRIPTION
**Pull request by nina-py**
## Goal

Allow the curators to add to or remove labels from existing collections.

-  Updated admin schema to add label external ids to mutation input.
- Updated DB-level resolver to add or replace existing labels to a collection and retrieve them as part of mutation response.
- Added integration tests.
- Added more explanatory comments to custom resolvers for label fields introduced in #1000.

## Reference

Tickets:

- https://getpocket.atlassian.net/browse/BACK-1651
- https://getpocket.atlassian.net/browse/BACK-1639

### CODE REVIEW

File: src/admin/resolvers/mutations/Collection.integration.ts [hunk 1]
```diff
@@ -439,13 +448,131 @@ describe('mutations: Collection', () => {
...
+        query: UPDATE_COLLECTION,
+        variables: {
+          data: input,
+        },
+      });
+
+      // make sure there are no errors before running other expect() statements
+      expect(data.errors).to.be.undefined;
+
+      // expect to see no labels whatsoever after the update
```
Comment (jpetto) [inline src/admin/resolvers/mutations/Collection.integration.ts:473]: tiny nit - looks like a copy pasta comment 😄

File: src/shared/resolvers/types.ts [hunk 2]
```diff
@@ -68,11 +68,17 @@ export const collectionLabelsFieldResolvers = {
...
+    // checks in place - if('externalId' in parent) rather than if(parent.externalId).
+
+    // The 'externalId' prop is present only on the Label entity, so if that exists,
+    // we can just return the value as is
     if ('externalId' in parent) {
       return parent.externalId;
     }
 
+    // The 'labelId' is present only on the CollectionLabel entity, so we need to
+    // query the linked Label entity to get the external ID of the label
```
Comment (jpetto) [inline src/shared/resolvers/types.ts:81]: 👏

**Review by jpetto — APPROVED** [review]
this is looking good! i think we can merge even with the failing spec test - we know that's only from the dev deployment/node cache.

File: src/admin/resolvers/mutations/Collection.integration.ts (hunk 1 above)
Comment (nina-py) [reply src/admin/resolvers/mutations/Collection.integration.ts]: Fixed!

**Review by jpetto — APPROVED** [review]

**Pull request merged**